README.md

```diff
@@ -50,3 +50,18 @@ sudo pip install -r requirements.txt
 sudo pip install -r dev_requirements.txt
 cp greenpithumb/wiring_config.ini.example greenpithumb/wiring_config.ini
 ```
+
+## QuickStart Instructions
+
+Run greenpithumb/greenpithumb.py and supply command-line flags for poll_interval, photo_interval, config_file, sleep_window, image_path, and db_file. Example below.
+
+```bash
+python greenpithumb/greenpithumb.py \
+  --poll_interval 15 \
+  --photo_interval 240 \
+  --config_file "C:\GreenPiThumb\greenpithumb\config.ini" \
+  --sleep_window "23:15-07:00" \
+  --sleep_window "14:30-15:45" \
+  --image_path "C:\GreenPiThumb\greenpithumb\images" \
```

Comment:
They'll be running this from the Pi, so it'll be Linux-style paths. Suggest just putting `"images/"`